src/App.tsx

```diff
@@ -1,10 +1,32 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [key, setPressedKey] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  useEffect(() => {
+    const handleKeyDown = (e: KeyboardEvent) => {
```

Comment:
Consider renaming 'handleKeyDown' to 'handleKeyUp' to match the event being handled.
![image](https://github.com/mate-academy/react_keyboard-hooks/assets/99403198/7ca79b14-2538-42ea-85dd-52f31e90413a)